packages/jest-expo-enzyme/src/setupEnzyme.native.ts

```diff
@@ -7,6 +7,15 @@ import Adapter from 'enzyme-adapter-react-16';
 import Enzyme from 'enzyme';
 import serializer from './serializer';
 
+type Global = {
```

Comment:
Should be possible in a simple way to get these defined with tsconfig.json via `@types`